app/src/components/v-form/v-form.vue

```diff
@@ -54,6 +54,7 @@
 				:batch-active="batchActiveFields.includes(field.field)"
 				:primary-key="primaryKey"
 				:loading="loading"
+				:values="values"
```

Comment:
Remove and use `inject`

Reply:
Actually this you can just remove. v-form is actually the component that provides the values object